packages/loader/container-loader/src/container.ts

```diff
@@ -1985,7 +1995,7 @@ export class Container
 		if (summary.details === undefined) {
 			summary.details = {};
 		}
-		summary.details.includesProtocolTree = this.options.summarizeProtocolTree === true;
```

Comment:
Here is where `Container` was counting on the `ContainerStorageAdapter` to overwrite the option upon service connect.